frontend/src/components/EditMedia.vue

```diff
@@ -40,12 +40,12 @@
       <o-button @click="$emit('close')">Cancel</o-button>
       <div>
         <o-button variant="primary" @click="save">
-          <action-button :mode="loading ? 'loading' : ''">Save</action-button>
+          <action-button :mode="loading ? 'loading' : 'off'">Save</action-button>
```

Comment:
i don't really get the `TestingMode` enum in `frontend/src/components/ActionButton.vue` but it seems like we have to explicitly set it to `"off"`?

Reply:
yeah, misguided attempts at bringing typescript into the repo. thanks for fixing!